src/multivariate/mvnormal.jl

```diff
@@ -256,7 +256,7 @@ sqmahal(d::MvNormal, x::AbstractVector) = invquad(d.Σ, broadcast(-, x, d.μ))
 sqmahal!(r::AbstractVector, d::MvNormal, x::AbstractMatrix) =
     invquad!(r, d.Σ, broadcast(-, x, d.μ))
 
-gradlogpdf(d::MvNormal, x::AbstractVector) = -(d.Σ \ broadcast(-, x, d.μ))
+gradlogpdf(d::MvNormal, x::AbstractVector{<:Real}) = -(d.Σ \ x .- d.μ)
```

Comment:
I suspect that this needs bracketing. i.e. `-(d.Σ \ (x .- d.μ))`

Reply:
fixed :)